Firestore/Source/Local/FSTLevelDB.h

```diff
@@ -69,6 +69,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 + (const leveldb::ReadOptions)standardReadOptions;
```

Comment:
These can be removed.

Reply:
The read options are still in use. I removed the write options.